test/kpod_run.bats

```diff
@@ -122,16 +122,13 @@ IMAGE="docker.io/library/fedora:latest"
 }
 
 @test "kpod run with volume flag" {
-    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} run -v ${MOUNT_PATH}:/run/test ${FEDORA_MINIMAL} cat /proc/self/mountinfo | grep '${MOUNT_PATH} /run/test rw,relatime'"
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} run -v ${MOUNT_PATH}:/run/test ${BB} cat /proc/self/mountinfo | grep '${MOUNT_PATH} /run/test rw,relatime'"
```

Comment:
Can we get one or two tests of basic volume handling here? No `:ro` or `:shared`, just `kpod mount -v /home` or similar

Reply:
@mheon that is the first test, there is no options just basic volume mounting.

Reply:
@umohnani8 That tests host:destination, but I didn't see a test for just `-v host` with no colons

Reply:
I believe we are not supporting just host, @rhatdan can you confirm.

Reply:
If that's true, LGTM

Reply:
Yes we are not supporting builtin volumes at least to start.  I would like to avoid them unless users insist.